tests/test_metas.py

```diff
@@ -1,9 +1,9 @@
-#  Copyright © 2020 Ingram Micro Inc. All rights reserved.
-
-import pytest
+#  Copyright © 2021 Ingram Micro Inc. All rights reserved.
```

Comment:
Copyright shall be 2020 - 2021, updaring last on each change of year afaik

Reply:
Will discuss and change in a separate task.